### DISCUSSION TITLE
Transformation tool bug

### DISCUSSION THREAD

**Discussion by wandrinsheep** [keiwando/toonsquid#53] [General]
So the transformation tool is very hard to move, I’ve pinpointed it to be a stage manager interaction causing it as it doesn’t happen while it off. But when windowed in stage manager I can select the anchors  maybe 4/10 attempts

**Comment by keiwando** [comment]
I just tried it with stage manager enabled in windowed mode but didn't notice any issues. Could you maybe record a video of it so that I can see what exactly you're experiencing?
